xc/fence/dune

```diff
@@ -0,0 +1,10 @@
+(executable 
+  (name fence)
+  (public_name fence.bin)
+  (package xapi-xenopsd-xc)
+  (flags -warn-error +a-3)
+  (libraries 
+    xenctrl
+  )
+)
+
```

Comment:
There is now a `dune unstable-fmt` command, do you think it'd be worth testing it on these dune files and submit improvements upstream? (both of us had our own jbuild reformatters)